app/src/test/java/com/duckduckgo/app/survey/api/SurveyDownloaderTest.kt

```diff
@@ -117,37 +115,6 @@ class SurveyDownloaderTest {
         verify(mockDao).deleteUnusedSurveys()
     }
 
-    @Test
-    fun whenAppTpSurveyContainsRetentionStudyControlSurveyAndUserInRetentionStudyThenSurveyScheduled() {
-        val mockAppTPCall = mock<Call<SurveyGroup?>>()
-        whenever(mockAppTPCall.execute()).thenReturn(Response.success(surveyWithAllocationForAppTPRetentionStudy("abc")))
```

Comment:
`surveyWithAllocationForAppTPRetentionStudy` function can be removed.

We may need to clean up SurveyGroup.SurveyOption as I noticed inside `surveyWithAllocationForAppTPRetentionStudy` we create a SurveyOption with `isAtpRetentionStudyRequired` which is not needed anymore.

Reply:
Yup, will do!